pkg/cluster/utils/utils.go

```diff
@@ -0,0 +1,89 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/redhat-developer/app-services-cli/pkg/cluster"
+	"github.com/redhat-developer/app-services-cli/pkg/localize"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+)
+
+func ResourceExists(ctx context.Context, c *cluster.KubernetesCluster, path string, serviceName string, opts cluster.Options) error {
```

Comment:
Utils should be used across the board and form SDK. Comments are strongly required to understand what this does without checking code.

Reply:
What is `Resource` here? This function needs to be documented.

Reply:
This returns an error but it should return a bool, or a bool + error.

Reply:
Sticking to err as adding a bool seems bit redundant.